### ISSUE TITLE
CommandBox Command?

### ISSUE THREAD

**Issue by lmajano** [foundeo/cfmlstats#3]
Can this be created as a commandbox command? So you can do something like:

`cfmlstats {dir}`

**Comment by pfreitag** [comment]
Yes it could certainly be turned into a command, the hardest part would be just formatting the output for the command line.
